javascript/javascript1/week3/js-week3.js

```diff
@@ -0,0 +1,96 @@
+// *Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+const removeAhmed = names.filter((e) => e.names === "Ahmed");
+names.splice(1, 1);
+console.log(names);
```

Comment:
The one you wrote is okay, but normally we use functions for this purpose. Here you are hard corded the name Ahmed to remove. So if you want to remove Samuel you need to repeat the same steps again, which is not good. So instead of that we use functions. For example
```
function removeNameFromNames(nameToRemove) {
  const indexOfNameToRemove = names.indexOf(nameToRemove);
  names.splice(indexOfNameToRemove, 1);
  console.log(names);
}

removeNameFromNames(Ahmed);
removeNameFromNames(Samuel);
```